briefcase/app.py

```diff
@@ -157,11 +157,16 @@ def find_support_pkg(self):
                             self.platform
                         )):
             for item in page.get('Contents', []):
-                candidates.append(item['Key'])
-        try:
-            return S3_URL + sorted(candidates, reverse=True)[0]
-        except IndexError:
-            return None
+                build_number = int(
+                    item['Key'].rstrip('.tar.gz').split('.')[-1].lstrip('b'))
+                if build_number > top_build_number:
+                    top_build_number, top_build = build_number, item['Key']
```

Comment:
Similarly - no reason for this to be a multi-assign.

Reply:
Sure sure.